software/Makefile

```diff
@@ -0,0 +1,41 @@
+SDK_DIR=../cosim/import/sdk
+NBF_DIR=../cosim/import/black-parrot/bp_common/software/py
+
+riscv-binaries=$(shell find ../. -iname "*.riscv")
+
+$(warning $(riscv-binaries))
+
+nothing:
+
+build_tools:
+	cd ../cosim/import; git submodule update --init sdk
+	make -j 4 -C $(SDK_DIR) panic_room
+
+build_software:
+	cd $(SDK_DIR); git submodule update --init bedrock
+	make -C $(SDK_DIR) -j1 bedrock
+	cd $(SDK_DIR); git submodule update --init perch
+	make  -C $(SDK_DIR) -j1 perch
+	cd $(SDK_DIR); git submodule update --init bootrom
+	make  -C $(SDK_DIR) -j1 bootrom
+	cd $(SDK_DIR); git submodule update --init beebs
+	make -C $(SDK_DIR) beebs
+
+clean_software:
+	-rm -rf $(SDK_DIR)/install/touchfiles/bedrock.*
+	-rm -rf $(SDK_DIR)/install/touchfiles/bootrom.*
+	-rm -rf $(SDK_DIR)/install/touchfiles/perch.*
+	-rm -rf $(SDK_DIR)/install/touchfiles/beebs.*
+	make -C $(SDK_DIR) prog_clean
+	make -C $(SDK_DIR) tidy_progs
+	-rm -rf nbf
+
+%.nbf: %.riscv
+	$(SDK_DIR)/install/riscv64-unknown-elf-dramfs/bin/objcopy -O verilog $< $*.mem
```

Comment:
$(SDK_DIR)/install/bin/riscv64-unknown-elf-dramfs-objcopy

Reply:
The other is temporary and not guaranteed to exist in an installation

Reply:
fixed